packages/ui-selectable/src/Selectable/index.tsx

```diff
@@ -54,61 +152,17 @@ tags: autocomplete, typeahead, combobox, dropdown, search
 **/
 class Selectable extends Component<Props> {
```

Comment:
It won't do anything for now, but could you please add the `@tsProps` flag to the description?
```
/**
---
category: components
tags: autocomplete, typeahead, combobox, dropdown, search
---
@tsProps
**/
```

This will tell the docs app to use the types for the docs page properties, once this PR is merged: https://github.com/instructure/instructure-ui/pull/642

Reply:
done